chainerx_cc/chainerx/python/routines.cc

```diff
@@ -554,6 +554,20 @@ void InitChainerxManipulation(pybind11::module& m) {
           "a"_a,
           "axis1"_a,
           "axis2"_a);
+    m.def("repeat",
+          [](const ArrayBodyPtr& a, int64_t repeats, absl::optional<int8_t> axis) {
+              return MoveArrayBody(Repeat(Array{a}, repeats, axis));
+          },
+          py::arg("a"),
+          py::arg("repeats"),
+          py::arg("axis") = nullptr);
+    m.def("repeat",
+          [](const ArrayBodyPtr& a, const std::vector<int64_t>& repeats, absl::optional<int8_t> axis) {
+              return MoveArrayBody(Repeat(Array{a}, repeats, axis));
+          },
+          py::arg("a"),
+          py::arg("repeats"),
+          py::arg("axis") = nullptr);
```

Comment:
How about using the `_a` literal here too?

Reply:
I fixed.